src/components/I18nDropdown.tsx

```diff
@@ -34,12 +34,12 @@ const I18nDropdown: FC = () => {
         id="lngSelect"
         ref={selectRef}
         onChange={dropdownHandler}
-        className="w-1/2 border bg-white dark:text-black xl:w-1/3"
+        className="w-auto border bg-white dark:text-black"
       >
         {langs.map((lang) => {
           return (
             <option key={lang} value={lang}>
-              {lang}
+              {t(`language.${lang}`)}
```

Comment:
What would you think if we add the language code as well?

So it would be "English (en)"

Would make portugese shorter (Portuguese (pt_BR) instead of "Portuguese (Brazilian)").

Reply:
What do you think @escapedcat ? :)

Reply:
@cstenglein Yes I understand. I would write it out like Portuguese (Brazilian) but we could fix the width of the dropdown in this specific usage. So you can read the full language when the dropdown is open but when you selected Portuguese (Brazilian) it is fixed. Probably (Brazilian) is not visible but I think that is not a problem.